src/mitol/google_sheets_refunds/api.py

```diff
@@ -80,10 +80,20 @@ def process_row(self, row_index, row_data):
                 result_type=ResultType.OUT_OF_SYNC,
                 message=None,
             )
-        result_type, message = self.hook.refunds_process_request(
-            refund_request_row=refund_req_row
-        )
-        if result_type == ResultType.PROCESSED:
+        results = self.hook.refunds_process_request(refund_request_row=refund_req_row)
+
+        failed = False
+        result_type = None
+        message = None
+
+        # walk all the results, if any did not suceed bail and we will return that result
+        for result_type, message in results:
```

Comment:
Does `refunds_process_request` return many results? We are only passing single row to it.